### DISCUSSION TITLE
When having an openapi plugin selected and having code with curly braces in chat history in a certain way, an f-string error is raised on the second user message

### DISCUSSION THREAD

**Discussion by Haherzke** [danny-avila/LibreChat#4818] [Troubleshooting]
### What happened?

When having a chat history that contains curly braces in a certain way, the next user query will not be answered and an error message is displayed from the llm. Steps to reproduce down below. Encountered this bug when using Librechat to help with coding  while having plugins enabled.

It only happens when using the plugin endpoint with at least one plugin selected. It doesn't matter if the LLM uses the plugin or not. Which LLM is selected doesn't matter.

Error in the Console: [handleAbortError] AI response error; aborting request: Single '}' in template.

Full error in the logs. The error seems to be raised because of the curly braces in an f-string function.

### Steps to Reproduce

1. Have your openAPI plugin via a json file in the .well-known folder
2. Select the plugin in a new chat
3. Enter following prompt: 
```
plugins {
id 'org.jetbrains.kotlin.jvm' version '1.8.0' // Beispielversion, passe sie an deine Version an
}
java {
toolchain {
languageVersion = JavaLanguageVersion.of(21) // Setze dies auf die gewünschte Java-Version
}
}
```
3. The answer of the LLM doesn't matter, just send another user message
5. The LLM will display an error message: "Something went wrong. Here's the specific error message we encountered: An error occurred while processing your request. Please contact the Admin."

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
{"level":"error","message":"[handleAbortError] AI response error; aborting request: Single '}' in template.","stack":"Error: Single '}' in template.\n    at Object.parseFString [as f-string] (node_modules\\@langchain\\core\\dist\\prompts\\template.cjs:51:19)\n    at parseTemplate (node_modules\\@langchain\\core\\dist\\prompts\\template.cjs:81:99)\n    at PromptTemplate.fromTemplate (node_modules\\@langchain\\core\\dist\\prompts\\prompt.cjs:93:41)\n    at HumanMessagePromptTemplate.fromTemplate (\\node_modules\\@langchain\\core\\dist\\prompts\\chat.cjs:289:56)\n    at createOpenAPIPlugin (\\api\\app\\clients\\tools\\dynamic\\OpenAPIPlugin.js:139:32)\n    at async constructorMap.<computed> (\\api\\app\\clients\\tools\\util\\loadSpecs.js:81:11)\n    at async loadTools (\\api\\app\\clients\\tools\\util\\handleTools.js:328:20)\n    at async PluginsClient.initialize (\\api\\app\\clients\\PluginsClient.js:109:18)\n    at async PluginsClient.sendMessage (\\api\\app\\clients\\PluginsClient.js:378:5)\n    at async \\api\\server\\routes\\ask\\gptPlugins.js:184:22"}
```

### Screenshots

![image](https://github.com/user-attachments/assets/bdcc9261-fd8c-47f6-947e-4e6ee689e5fa)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Plug-ins will be deprecated soon, I suggest you try agents: https://github.com/danny-avila/LibreChat/issues/3607
